src/minoversikt/huskelapp/modalvisning/SlettHuskelappKnapp.tsx

```diff
@@ -44,7 +44,7 @@ export const SlettHuskelappKnapp = ({
             bekreftelsesmelding={{
                 overskrift: 'Er du sikker på at du vil slette huskelappen?',
                 beskrivelse:
-                    'Huskelappen slettes, men kan utleveres ved innsynsbegjæring innenfor oppfølgingsperioden.',
+                    'Huskelappen slettes, men kan utleveres hvis personen ber om innsyn i denne oppfølgingsperioden.',
```

Comment:
```suggestion
                    'Huskelappen slettes, men kan utleveres hvis personen ber om innsyn i løpet av denne oppfølgingsperioden.',
```

For meg er det uklart om dei kan be om innsyn etter at perioden er over, foreslår å leggje til ein "i løpet av"

Reply:
Vi tenkte på et par forskjellige i går, men kom aldri fram til "i løpet av" 😅 Enig at det er tydeligere og mer riktig!